server/routes/route-helpers.ts/users-helpers.ts

```diff
@@ -0,0 +1,62 @@
+import { client } from "../../src/index";
+
+export const createUser = async (req: any, res: any) => {
+  await client
+    .query(
+      `
+        INSERT INTO users ("userName", "firebaseAuthId", "profileImgUrl")
+        VALUES ('${req.body.userName}', '${req.body.firebaseAuthId}', '${req.body.profileImgUrl}');
+      `
+    )
+    .then(() => res.send("you have successfully created a user"))
```

Comment:
change this to `res.status(200).send("you have successfully created a user")`.

to have a status code for whether it is successful or not. 200 is a standard code for success.

Reply:
[check out this link](https://www.dotcom-monitor.com/blog/2022/03/03/the-10-most-common-http-status-codes/#:~:text=Status%20Code%20404%20%E2%80%93%20The%20most,due%20to%20improper%20URL%20redirection.)